### PR TITLE
More fantomas

### PR DESCRIPTION
**Pull request by nojaf**
Related to https://github.com/dotnet/fsharp/issues/12119#issuecomment-1239237159

@dsyme I formatted some more files.
I've also expanded all glob patterns, this would allow for any new files in that folder to be formatted from day one.

Oh, and I updated to beta-009 which had a fix for https://github.com/fsprojects/fantomas/issues/2289, so some other files are impacted by that as well.

### CODE REVIEW

File: src/Compiler/Checking/import.fs
Comment (auduchinok) [inline src/Compiler/Checking/import.fs:161]: @nojaf Is this change expected?

**Comment by dsyme** [comment]
We should probably wait until https://github.com/dotnet/fsharp/pull/13737 is in as the conflicts here will be intense (I know we can pre-format that code to match the new settings, but probably easy just to wait for a merge

File: src/Compiler/Checking/import.fs
Comment (nojaf) [reply src/Compiler/Checking/import.fs:161]: I don't think so no, please open an issue for this.
Ref: https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-indexslice-expressions

**Comment by nojaf** [comment]
@vzarytovskii could you add `5.0.0-beta-009` to the feed please?

**Comment by vzarytovskii** [comment]
> @vzarytovskii Vlad Zarytovskii FTE could you add `5.0.0-beta-009` to the feed please?

Yep, made a request.

**Comment by nojaf** [comment]
Thanks!

**Comment by dsyme** [comment]
@nojaf OK now #13737 is in can you re-run the fantomas formatting on this?  Thanks

File: src/Compiler/Service/ServiceLexing.fs
Comment (dsyme) [inline src/Compiler/Service/ServiceLexing.fs:208]: This formatting is much better

**Comment by nojaf** [comment]
@dsyme, sure I can pick that up next week.
@vzarytovskii could you add [5.0.2](https://www.nuget.org/packages/fantomas/5.0.2) to the feed?

**Comment by vzarytovskii** [comment]
> @dsyme, sure I can pick that up next week.
> @vzarytovskii could you add [5.0.2](https://www.nuget.org/packages/fantomas/5.0.2) to the feed?
> 

Sure, likely gonna be added on Monday

**Comment by nojaf** [comment]
@dsyme I've expanded the glob patterns in the ignore file and updated Fantomas to the latest stable version. I would include new files in future PRs.

@vzarytovskii could you add `5.0.3`, please?

**Comment by vzarytovskii** [comment]
@nojaf 
> @vzarytovskii could you add `5.0.3`, please?

Created the request, will restart CI once it is there.

**Review by T-Gro — APPROVED** [review]
Good work - especially good to have all new files checked by default now that exclusions are explicit.

**Review by KevinRansom — APPROVED** [review]

**Comment by nojaf** [comment]
Hi @vzarytovskii, all good for you?

**Comment by vzarytovskii** [comment]
> Hi @vzarytovskii, all good for you?

Sure, should be auto merged once green. Thanks!

**Comment by nojaf** [comment]
Oh, I'm asking because of 
image

I think I still need your explicit approval.

**Comment by T-Gro** [comment]
> Oh, I'm asking because of image
> 
> I think I still need your explicit approval.

Should be good now, just the conversations had to be resolved. (I also marked the one from @auduchinok  as resolved as I think this PR still deserves to go in)
Now only waiting for CI checks and its merged automatically.

**Comment by dsyme** [comment]
Great work here, would love to see continued incremental work on applying fantomas to all code files.

**Pull request merged**
